scripts/compile_cost_assumptions.py

```diff
@@ -1639,6 +1687,9 @@ def add_mean_solar_rooftop(data):
     # add costs for gas pipelines
     data = pd.concat([data, costs_ISE.loc[["Gasnetz"]]], sort=True)
 
+    # add (enhanced) geothermal systems data
+    data = add_egs_data(data) 
```

Comment:
Great, that's perfect with the extra function

Reply:
😎